config/settings.yml.example

```diff
@@ -58,6 +58,11 @@
 # default values for https_port is 8443
 #:https_port: 8443
 
+# http server type configuration
+# A string that indicates the server type (possible values: 'webrick', 'puma').
+# Default value is 'webrick'
+#:http_server_type: 'puma'
```

Comment:
you're suggesting 'puma' but expecting `:puma` later in the code (when I tried 'puma', the server didn't even start

```
D, [2018-10-16T13:59:47.376959 ] DEBUG -- : ["/home/inecas/Projects/ws/foreman-rex/smart-proxy/lib/launcher.rb:172:in `block in add_server'", "/home/inecas/Projects/ws/foreman-rex/smart-proxy/lib/launcher.rb:172:in `each'", "/home/inecas/Projects/ws/foreman-rex/smart-proxy/lib/launcher.rb:172:in `add_server'", "/home/inecas/Projects/ws/foreman-rex/smart-proxy/lib/launcher.rb:192:in `block in launch'"]
```

Reply:
I know, already fixed that, didn't push it yet :)